### PR TITLE
HALO-20240919a plan and report

### PR DESCRIPTION
**Pull request by yoctoyotta1024**
- minor updates to HALO-20240919a flight plan
- added first draft of HALO-20240919a flight report

### CODE REVIEW

File: orcestra_book/plans/HALO-20240919a.md [hunk 1]
```diff
@@ -30,8 +30,8 @@ kernelspec:
   name: python3
 orphan: true
 platform: HALO
-takeoff: '2024-09-19 10:33:00Z'
-landing: '2024-09-19 19:33:00Z'
+takeoff: '2024-09-16 10:40:00Z'
+landing: '2024-09-16 18:40:00Z'
```
Comment (lkluft) [inline orcestra_book/plans/HALO-20240919a.md:34]: Are you sure about the date? ;)

Comment (yoctoyotta1024) [reply orcestra_book/plans/HALO-20240919a.md]: whoops thanks ;)

**Comment by yoctoyotta1024** [comment]
@lkluft can we add a link to the geos + flight track video you made? Maybe the one with the dropsonde markers?

**Comment by lkluft** [comment]
> @lkluft can we add a link to the geos + flight track video you made? Maybe the one with the dropsonde markers?

Simply add the following code snippet to embed the video (you can adjust the width to your liking).

```html
<video width="790" controls="" poster="https://swift.dkrz.de/v1/dkrz_948e7d4bbfbb445fbff5315fc433e36a/ORCESTRA/static/HALO-20240919a-thumb.png">
  <source src="https://swift.dkrz.de/v1/dkrz_948e7d4bbfbb445fbff5315fc433e36a/ORCESTRA/static/HALO-20240919a.mp4" type="video/mp4">
  Your browser does not support the video tag.
</video>
```

**Comment by yoctoyotta1024** [comment]
Ah absolutely so cool thank you!! Am I alright to wrap it in <center> </center> tags?

Also is the caption okay? I assume it's dropsonde launch detects not ground positions but other than they are GEOS I'm not sure what images exactly are being being shown...

File: orcestra_book/reports/HALO-20240919a.md [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+## Conditions
+
+<center>
+<video width="590" controls="" poster="https://swift.dkrz.de/v1/dkrz_948e7d4bbfbb445fbff5315fc433e36a/ORCESTRA/static/HALO-20240919a-thumb.png">
+  <source src="https://swift.dkrz.de/v1/dkrz_948e7d4bbfbb445fbff5315fc433e36a/ORCESTRA/static/HALO-20240919a.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+</center>
+
+_Flight path superimposed on the color image from NOAA's Geostationary Operational Environmental Satellites (GOES) 16 satellite on 19th September 2024. The green dots mark sucessful dropsonde launches. Credit: Lukas Kluft._
```
Comment (yoctoyotta1024) [inline orcestra_book/reports/HALO-20240919a.md:85]: (see here)

Comment (lkluft) [reply orcestra_book/reports/HALO-20240919a.md:85]: Technically, the layer is called "GOES-East ABI GeoColor", but our shortcut is just "visible" 😉

For the dropsondes, it is their actual location at a given time. You can actually see them move a bit for about 30 minutes. The drift is just so small that it looks stationary. But you can see that they all stop slightly west of the track due to the mean wind.

**Review by lkluft — CHANGES_REQUESTED** [review]
The report looks really nice in general!

There is one technical issue, which is the file size of the embedded pictures. Many of them are several MB, which is definitely too large. It is rather easy to resize them on the command line using ImageMagick:
```
magick big_image.jpg -resize 1000x resized_image.jpg.
```
Afterwards, you need to make sure to rebase the branch in a way that the original large images are not included anymore (e.g. squash the commit that adds the images and the one that resizes them).

Other than that the report looks good to be merged :)

**Review by lkluft — APPROVED** [review]

File: orcestra_book/reports/HALO-20240919a.md (hunk 2 above)
Comment (yoctoyotta1024) [reply orcestra_book/reports/HALO-20240919a.md]: Great thanks, I've updated the figure caption.

P.s. Oh yes sorry I'd forgotten you told me that about the dropsondes - they're even cooler than I thought :)

**Comment by yoctoyotta1024** [comment]
Oki doki I think I've fixed all the images so I'm happy for this to be merged if you are!

**Comment by lkluft** [comment]
Thanks for updating! :) Should we merge or a you expecting more feedback?

**Pull request merged**